cartridges/int_cloudinary/cartridge/scripts/model/cloudinaryModel.js

```diff
@@ -73,7 +74,8 @@ var getCloudinaryImages = function (productID, params) {
         if (cloudinaryConstants.CLD_ENABLED) {
             // initialize optional params
             if (!empty(params)) {
-                colorAttrValueID = params.variationAttrValueID;
+                colorAttrValueID = params.variationAttrValueID ? params.variationAttrValueID : params.variationColorAttrID;
```

Comment:
Is the argument name valid?

Reply:
@asad-rafter I mean "variationAttrValueID", which seems always the color variant